src/main/java/com/uber/jenkins/phabricator/BuildResultProcessor.java

```diff
@@ -130,6 +133,46 @@ public void processRemoteComment(String commentFile, String commentSize) {
     }
 
     /**
+     * Fetch remote lint violations from the build workspace and process
+     *
+     * @param conduit      conduit API client
+     * @param lintFile     the path pattern of the file
+     * @param lintFileSize maximum number of bytes to read from the remote file
+     */
+    public void processLintResults(ConduitAPIClient conduit, String lintFile, String lintFileSize) {
+        RemoteFileFetcher lintFetcher = new RemoteFileFetcher(workspace, logger, lintFile, lintFileSize);
+        try {
+            String input = lintFetcher.getRemoteFile();
+            if (input != null && input.length() > 0) {
+                lintResults = new LintResults();
+                BufferedReader reader = new BufferedReader(new StringReader(input));
+
+                String lint;
+                while ((lint = reader.readLine()) != null) {
+                    JSONObject json = JSONObject.fromObject(lint);
+                    lintResults.add(new LintResult(
+                            (String) json.get("name"),
+                            (String) json.get("code"),
+                            (String) json.get("severity"),
+                            (String) json.get("path"),
+                            (Integer) json.get("line"),
+                            (Integer) json.get("char"),
+                            (String) json.get("description")));
+
+                }
+                boolean pass = buildResult.isBetterOrEqualTo(hudson.model.Result.SUCCESS);
+                new PostLintTask(logger, conduit, phid, pass, unitResults, harbormasterCoverage, lintResults).run();
```

Comment:
This shouldn't be a separate conduit call. we should send the lint results along with the unit/coverage data